src/ts/useSnackBar.tsx

```diff
@@ -0,0 +1,34 @@
+import { AxiosError } from 'axios';
+import { create } from 'zustand';
+
+interface SnackBar {
+  showSnackBar: boolean,
+  message: string,
+  open: () => void,
+  close: () => void,
+  setMessage: (errorMessage: string) => void
+}
+
+interface AxiosErrorMessage {
+  message: string
+}
+
+export const useStore = create<SnackBar>((set) => ({
+  showSnackBar: false,
+  message: '',
+  open: () => set({ showSnackBar: true }),
+  close: () => set({ showSnackBar: false }),
+  setMessage: (message: string) => set({ message }),
+}));
+
+export const useSnackBar = () => {
+  const { open, setMessage } = useStore();
+  const onError = (e: unknown) => {
+    const err = e as AxiosError;
```

Comment:
```suggestion
  const onError = (e: AxiosError) => {
```
이런식으로 e의 타입의 좀 더 엄격하게 처리하는 것은 어떤가요?

Reply:
다음 브랜치에서 작업하겠습니다!